src/Utils/SqlStatementKeys.php

```diff
@@ -87,98 +87,4 @@ class SqlStatementKeys extends \TechDivision\Import\Product\Utils\SqlStatementKe
      * @var string
      */
     const URL_REWRITE_PRODUCT_CATEGORIES_BY_SKU = 'ur_rewrite_product_categories.by.sku';
-
-    /**
-     * The SQL statements.
-     *
-     * @var array
-     */
-    private $statements = array(
-        SqlStatementKeys::CREATE_URL_REWRITE =>
-            'INSERT
-               INTO url_rewrite
-                    (entity_type,
-                     entity_id,
-                     request_path,
-                     target_path,
-                     redirect_type,
-                     store_id,
-                     description,
-                     is_autogenerated,
-                     metadata)
-            VALUES (:entity_type,
-                    :entity_id,
-                    :request_path,
-                    :target_path,
-                    :redirect_type,
-                    :store_id,
-                    :description,
-                    :is_autogenerated,
-                    :metadata)',
-        SqlStatementKeys::UPDATE_URL_REWRITE =>
-            'UPDATE url_rewrite
-                SET entity_type = :entity_type,
-                    entity_id = :entity_id,
-                    request_path = :request_path,
-                    target_path = :target_path,
-                    redirect_type = :redirect_type,
-                    store_id = :store_id,
-                    description = :description,
-                    is_autogenerated = :is_autogenerated,
-                    metadata = :metadata
-              WHERE url_rewrite_id = :url_rewrite_id',
-        SqlStatementKeys::CREATE_URL_REWRITE_PRODUCT_CATEGORY =>
-            'INSERT
-               INTO catalog_url_rewrite_product_category
-                    (url_rewrite_id,
-                     category_id,
-                     product_id)
-             VALUES (:url_rewrite_id,
-                     :category_id,
-                     :product_id)',
-        SqlStatementKeys::UPDATE_URL_REWRITE_PRODUCT_CATEGORY =>
-            'UPDATE catalog_url_rewrite_product_category
-                SET category_id = :category_id,
-                    product_id = :product_id
-              WHERE url_rewrite_id = :url_rewrite_id',
-        SqlStatementKeys::DELETE_URL_REWRITE_PRODUCT_CATEGORY =>
-            'DELETE
-               FROM catalog_url_rewrite_product_category
-              WHERE url_rewrite_id = :url_rewrite_id',
-        SqlStatementKeys::URL_REWRITE_PRODUCT_CATEGORY =>
-            'SELECT *
-               FROM catalog_url_rewrite_product_category
-              WHERE url_rewrite_id = :url_rewrite_id',
-        SqlStatementKeys::URL_REWRITES_BY_SKU =>
-            'SELECT t2.*
-               FROM catalog_product_entity t1,
-                    url_rewrite t2
-              WHERE t1.sku = :sku
-                AND t2.entity_id = t1.entity_id
-                AND t2.entity_type = \'product\'',
-        SqlStatementKeys::URL_REWRITE_PRODUCT_CATEGORIES_BY_SKU =>
```

Comment:
This is not available in 'TechDivision\Import\Utils\SqlStatements' and can therefor not been deleted.